charts/vela-core/values.yaml

```diff
@@ -110,10 +110,12 @@ optimize:
   enableResourceTrackerDeleteOnlyTrigger: true
 
 ##@param featureGates.enableLegacyComponentRevision if disabled, only component with rollout trait will create component revisions
-##@param featureGates.gzipResourceTracker if enabled, resourceTracker will be compressed before stored
+##@param featureGates.gzipResourceTracker if enabled, resourceTracker will be compressed using gzip before being stored
+##@param featureGates.zstdResourceTracker if enabled, resourceTracker will be compressed using zstd before being stored
 featureGates:
   enableLegacyComponentRevision: false
   gzipResourceTracker: false
+  zstdResourceTracker: false
```

Comment:
If users enable both the gzip and zsyd not carefully, it is strange. How are priorities determined?

Reply:
`zstd` have higher priority.

Reply:
Add some comments about this in the values file.

Reply:
Good idea. Sure